### ISSUE TITLE
No module named web3

### ISSUE THREAD

**Issue by AAZZ123** [Yapote/ARBITRUM-SNIPING-BOT-ARBX#7]
from web3 import Web3 as Q
ModuleNotFoundError: No module named 'web3'

i've got this same error for all your bot, i've searching 5 hours and uninstall py, install again.. do the same and search i don't found the solution !

**Comment by Yapote** [comment]
Just type "pip3 install web3" and that should do it. Otherwise you are opening the terminal in your root and not in the bot folder

**Comment by AAZZ123** [comment]
I tried to install with Visual studio code once, and same error. and another time with git bash.. and i don't know why it's still say the same error..

**Comment by Yapote** [comment]
It's because you are not opening the terminal in the folder so the bot can't find it

**Comment by AAZZ123** [comment]
![arbibut](https://user-images.githubusercontent.com/35351379/228247918-98580c51-4588-4355-a669-073f1cf1b8a4.png)

**Comment by AAZZ123** [comment]
![thisone](https://user-images.githubusercontent.com/35351379/228266839-ddfda246-7e6c-4145-bf72-4605a6f8b190.png)
I already had this type of error yesterday, and I installed visual build tool. Now I have this that appears but I have the impression that it has nothing to do with web3

**Comment by Yapote** [comment]
Yes that is correct, https://bobbyhadz.com/blog/python-error-legacy-install-failure

**Comment by Yapote** [comment]
Okay :) But I just want to tell you, if you want to make it big, you need to sweat. This bot made me personally 7k USD just in ARB airdrops

**Comment by AAZZ123** [comment]
I I understand, but yesterday I spent 5 hours searching before coming to github. Now 3 hours to install lru-dict, trying your link + what I can find on google and it's the same I do not understand.

**Comment by Yapote** [comment]
Okay you know what. I will compile it for you so you don't need to install anything when I get some free time today. Are you running Windows?

**Comment by AAZZ123** [comment]
Thanks bro but no it would mean that if I have to install something else later that will use these packages it still won't work, I would want it to be installed properly. i gonna uninstall python, visual build, and git clone the folder again and if it doesn't work it's bad luck !
